test/commits.js

```diff
@@ -222,6 +222,19 @@ describe('getMerge', () => {
         href: 'https://gitlab.com/user/repo/merge_requests/15007'
       })
     })
+
+    it('parses a merge for subgroups', () => {
+      const message = 'Merge branch \'branch\' into \'master\'\n\nMemoize GitLab logger to reduce open file descriptors\n\nCloses gitlab-ee#3664\n\nSee merge request user/repo/subgroup!15007'
+      const remote = {
+        hostname: 'gitlab.com',
+        url: 'https://gitlab.com/user/repo/subgroup'
+      }
+      expect(getMerge(message, remote)).to.deep.equal({
+        id: '15007',
+        message: 'Memoize GitLab logger to reduce open file descriptors',
+        href: 'https://gitlab.com/user/repo/subgroup/merge_requests/15007'
```

Comment:
@a-ignatov-parc is this valid/realistic URL for a subgroup merge request? I've updated the test, but I've never used gitlab subgroups. I want to keep the tests close to what is likely in the wild.

Reply:
Yes. Here is a real link: `https://gitlab.rambler.ru/unity/sites/wmj/merge_requests/193`

Reply:
Btw thank for helping 😉